### ISSUE TITLE
Configuration files with dash are ignored

### ISSUE THREAD

**Issue by tkzv** [containous/traefik#4645]
If configuration file is set in the command line with "-c" option and the file name contains a dash (hyphen, minus sign) "-", Traefik will use the default configuration. It fails silently, with no messages that the file was ignored; it's only noticeable by its behaviour.

For example,
```
./traefic -c config.toml 
```
correctly loads the configuration in config.toml, but 
```
./traefic -c config-2.toml 
```
uses default settings and ignores the configuration file.

Observed in the latest 2.0.0 alpha in Windows and Linux:
```
$ ./traefik version
Version:      2.0.0-alpha1
Codename:     faisselle
Go version:   go1.12.1
Built:        2019-03-18T14:26:35Z
OS/Arch:      linux/amd64

>traefik.exe version
2019/03/21 13:51:55 Using high precision timer
Version:      2.0.0-alpha1
Codename:     faisselle
Go version:   go1.12.1
Built:        2019-03-18T14:40:53Z
OS/Arch:      windows/amd64
```

Example configuration:
```toml
[global]
sendanonymoususage = false
```
If the configuration file is called "config-2.toml", there will be a message "You haven't specify the sendAnonymousUsage option, it will be enable by default."

Workaround:
```
./traefik --configfile=config-2.toml
```
works correctly

**Comment by ldez** [comment]
hello, when you have a value that contains a hyphen, you have to add quotes around the value.

```
./traefic -c "config-2.toml"
```

>  It fails silently, 

related to https://github.com/containous/traefik/issues/2927

PS: could you follow the issue template.

**Comment by tkzv** [comment]
> PS: could you follow the issue template.
Do you want me to rewrite the bug with the template?

**Comment by ldez** [comment]
Sorry it's:
```
./traefic --config="config-2.toml"
```

**Comment by tkzv** [comment]
I wrote above that --config= works even without quotes. The bug is that -c does not work with dashes. Regardless of the quotes.

**Comment by ldez** [comment]
yes short flag doesn't works with dash, it's a standard behavior.

**Comment by tkzv** [comment]
Why? Never seen anything like that.

**Comment by traefiker** [comment]
Closed by #4935.
